### DISCUSSION TITLE
Welcome to MooTool Discussions!

### DISCUSSION THREAD

**Discussion by rememberber** [rememberber/MooTool#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tseenola** [comment]
I feel that Notes is very good, but I have two suggestions:
It would be great if the font size in the content of Notes could be cached. Otherwise, every time we create a new Note, we need to adjust the font size again.
Can the file names in Notes be renamed?

**Reply by rememberber** [reply]
感谢支持！重命名，最新版本已经支持了，右键菜单，欢迎更新使用
